### ISSUE TITLE
Side navigation does not recieve keyboard focus

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/rockarch.org#72]
## Describe the bug
On pages with a side navigation, such as https://rockarch.org/collections/access-and-request-materials/, the menu items do not receive keyboard focus because the tabindex property is set to `-1`

## Expected behavior
The side navigation menu items should receive visible focus as part of the page's tab order.

## Impact on your work
The side navigation is not accessible using keyboard navigation
